include/aos/sm/config.hpp

```diff
@@ -71,4 +71,18 @@
 #define AOS_CONFIG_NETWORKMANAGER_MAX_NUM_EXPOSED_PORTS 8
 #endif
 
+/**
+ * Max exposed port length.
+ */
+#ifndef AOS_CONFIG_NETWORKMANAGER_EXPOSED_PORT_LEN
```

Comment:
Do we need it configured? If not you can specify only const:

cExposedPortLen = cProtocolNameLen + cPortLen

Reply:
Done